x-pack/test_serverless/README.md

```diff
@@ -60,7 +60,7 @@ added like this to a test suite:
 
 ```
 describe('my test suite', function () {
-  this.tags(['skipSvlOblt', 'skipSvlSearch', 'skipSvlSec']);
+  this.tags(['skipSvlOblt', 'skipSvlSearch', 'skipSvlSec']); // SKIPS ON ALL ENVIRONMENTS, INCLUDING MKI
```

Comment:
I got confused here: what are the `ENVIRONMENTS`?
_idea_: To make sure everyone understands, maybe we can explicitly mention that it will skip tests in both regular Kibana serverless pipelines that start Kibana in serverless mode against serverless ES node, and pipelines creating serverless projects in MKI (Cloud)

Reply:
Yeah I like that. 

This came about as I was confused between `this.tags(['skipSvlOblt', 'skipSvlSearch', 'skipSvlSec']);` and `this.tags(['failsOnMKI']);`

The former applies to all serverless (or one project type), while the latter is all environments(projects).  

I'll fixup